centrallix-lib/src/net_raw_api.c

```diff
@@ -192,7 +192,7 @@ rawSendParam(pNetManagedConn conn, char param_type, pObjData value, int cmd_id)
 		break;
 
 	    case NET_PARAM_MONEY:
-	        memcpy(buf+1, value->Money, 6);
+	        memcpy(buf+1, value->Money, 8);
 		len = 7;
```

Comment:
It is setting up the buffer which will be written with fdWrite later on line 210.  I changed from 6 to 8 because old representation was int+unsigned short and new is long long.  I also now changed len to 9, because len is size+1 in all the cases; this change will show in my next push.